docs/40_development.md

```diff
@@ -144,3 +144,15 @@ dev team sees them. However, these emails arrive with a long, spammy-looking
 From address with a Heroku DNS domain; this seems to be an artifact of how
 mit.edu processes emails, and does not occur in general (e.g. messages sent
 from the API server to users).
+
+### Use a-sentence-per-line formatting in .md/.rst files
+
+One sentence per line should be entered instead of wrapping into paragraphs or making one paragraph per line.
+Such a convention is used by a number of projects and advocated by others (see, e.g., [nick.groenen's post](https://nick.groenen.me/notes/one-sentence-per-line/)) since it:
```

Comment:
I don't think I will add it here, but found in https://github.com/executablebooks/mdformat/blob/782cb22f294bc4ce48368ee173de4837fc208ecf/docs/users/style.md?plain=1#L159 the quote from "Brian W. Kernighan. "UNIX for Beginners". 1974" which mentions "Start each sentence on a new line.", although in the sample paragraph quoted there this particular sentence doesn't because previous one was wrapped into it already ;-)